ansible/cassandra-exporter.yaml

```diff
@@ -1,4 +1,4 @@
-- hosts: cassandra
+- hosts: cassandra-ps
```

Comment:
why cassandra-ps ?

Reply:
Can you please name it as cassandra-monitoring-hosts. It'll make the group name self explanatory.

Reply:
Updated the host name as mentioned @rjshrjndrn